nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -87,6 +87,8 @@
 
 ## New Modules {#sec-release-24.11-new-modules}
 
+- [hardware.block.iosched](options.html#hardware-block-iosched.defaultScheduler) allows configuration of I/O schedulers for block devices.
```

Comment:
Isn't it just `hardware.block`? According to the module below `hardware.block.iosched` doesn't exist.

Reply:
Oops. Corrected it.